codersdk/users.go

```diff
@@ -90,6 +90,7 @@ type LicensorTrialRequest struct {
 type CreateFirstUserRequest struct {
 	Email     string                   `json:"email" validate:"required,email"`
 	Username  string                   `json:"username" validate:"required,username"`
+	FullName  string                   `json:"name" validate:"user_real_name"`
```

Comment:
Why do we need a special validation for this? IMO we can use something like`validate:min=3,max=50`

Reply:
I'm not sure why the original validation was added, but I'm using it here for consistency with `UpdateUserProfile`

Reply:
I feel like we're ending up with a bit of inconsistency in the API and API responses. There's a disconnect between full name and name since we're calling it one thing in some places and something else in other places. Here we're essentially:

1. Reading in "full name" (cli)
2. Serializing as "name" over the wire (JSON)
3. Deserializing as "full name"
4. Writing "name" to database 😅